src/Catty/Helpers/Util/Util.swift

```diff
@@ -50,6 +50,34 @@ func synchronized(lock: AnyObject, closure: () -> Void) {
         return appBuildVersion
     }
 
+    class func allMessages(forProject project: Project) -> NSMutableOrderedSet? {
```

Comment:
I would suggest to make this set non-optional (return empty set if no messages) and to make this set immutable since adding items to this set has no effect and thus should not be possible.

Please also consider making a more Swift-like name (while the Objective C name is perfect, `@objc`), .e.g., `allMessages(for project: Project)`.

Reply:
I've made the return value a non-optional `NSOrderedSet` following your suggestions.

Reply:
Thanks! Could you please also look at the method signature, i.e., the parameter name as suggested?